data/objects/tom.urdf

```diff
@@ -10,32 +10,18 @@
                 <mesh filename="tom.dae" scale="0.8 0.8 0.8"/>
             </geometry>
         </visual>
-        <!-- <visual>
-	    <origin xyz="0 0 0" />
+        <visual>
```

Comment:
I removed the other collision mesh, but I think it is useful to have a semi-transparent orange collision mesh here, since it gives us an idea of where the user's head is in relation to it.

Reply:
And it doesn't obfuscate the visual meshes, unlike with the wheelchair.